cypress-tests/cypress/e2e/RoutingTest/00000-PriorityRouting.cy.js

```diff
@@ -0,0 +1,209 @@
+import * as fixtures from "../../fixtures/imports";
+import State from "../../utils/State";
+import * as utils from "../RoutingUtils/utils";
+
+let globalState;
+
+describe("Priority Based Routing Test", () => {
+  let should_continue = true; // variable that will be used to skip tests if a previous test fails
+
+  beforeEach(function () {
+    if (!should_continue) {
+      this.skip();
+    }
+  });
+
+  before("seed global state", () => {
+    cy.task("getGlobalState").then((state) => {
+      globalState = new State(state);
+    });
+  });
+
+  afterEach("flush global state", () => {
+    cy.task("setGlobalState", globalState.data);
+  });
+
+  context("Routing with Stripe as top priority", () => {
+    it("create-jwt-token", () => {
+      let data = utils.getConnectorDetails("common")["jwt"];
+      let req_data = data["Request"];
+      let res_data = data["Response"];
+
+      cy.createJWTToken(req_data, res_data, globalState);
+      if (should_continue)
+        should_continue = utils.should_continue_further(res_data);
+    });
+
```

Comment:
Can this be moved to top level describe just so that we don't need to login again and again?